community/cypher/cypher-compiler-3.2/src/test/scala/org/neo4j/cypher/internal/compiler/v3_2/spi/QueryContextAdaptation.scala

```diff
@@ -32,6 +32,8 @@ trait QueryContextAdaptation {
 
   override type EntityAccessor = this.type
 
+  override def resources: CloseableResource = ???
```

Comment:
Basically almost all the methods in that class are unimplemented, moreover that class is used only in tests. I don't think there is any need to add any explanation.